go.sum

```diff
@@ -1,11 +1,18 @@
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
```

Comment:
The hyperloglog github README headlines "use metro hash instead of xxhash", but apparently they still use it somewhere. 🤷‍♂

Reply:
Yeah and I'm also not sure why they need to import the entirety of influxdb as well.